src/text/text.rs

```diff
@@ -565,27 +570,27 @@ mod tests {
 
         let style_top = TextStyleBuilder::new()
             .character_style(character_style)
-            .vertical_alignment(VerticalAlignment::Top)
+            .baseline(Baseline::Top)
             .build();
-        let style_center = TextStyleBuilder::new()
+        let style_middle = TextStyleBuilder::new()
             .character_style(character_style)
-            .vertical_alignment(VerticalAlignment::Center)
+            .baseline(Baseline::Middle)
             .build();
         let style_bottom = TextStyleBuilder::new()
             .character_style(character_style)
-            .vertical_alignment(VerticalAlignment::Bottom)
+            .baseline(Baseline::Bottom)
             .build();
         let style_baseline = TextStyleBuilder::new()
             .character_style(character_style)
-            .vertical_alignment(VerticalAlignment::Baseline)
+            .baseline(Baseline::Alphabetic)
             .build();
 
         Text::new("t", Point::new(0, 8))
             .into_styled(style_top)
             .draw(&mut display)
             .unwrap();
-        Text::new("c", Point::new(6, 8))
-            .into_styled(style_center)
+        Text::new("m", Point::new(6, 8))
```

Comment:
Nit: I'm not sure if changing "c" to "m" has a purpose. Since otherwise it would just inflate the diff, I'm assuming it has, I just can't figure out what. Could you please explain? :)

Reply:
It was on purpose: `VerticalAlignment::Center` was renamed to `Baseline::Middle`.

Reply:
I did not at all manage to associate the letters with the alignments, lol.